Makefile

```diff
@@ -18,8 +19,9 @@ endif
 include $(SHIPYARD_DIR)/Makefile.inc
 
 TARGETS := $(shell ls -p scripts | grep -v -e /)
-export LDFLAGS = -X main.VERSION=$(VERSION)
```

Comment:
This was setting main.VERSION, which is:

```
107     VERSION            = "not-compiled-properly"                                                                                                                  
```

Which is also used in:

```
185     components.cableEngineSyncer = syncer.NewGatewaySyncer(
186         components.cableEngine,
187         submarinerClient.SubmarinerV1().Gateways(components.submSpec.Namespace),
188         VERSION, components.cableHealthChecker)                                                                                                                   
```

The new code added does not set version.version anywhere, so now we always
log "Submariner Version: devel" and use "not-compiled-properly" in the call above.

This change should:
- Replace LDFLAG "-X main.VERSION=$(VERSION)" with "-X github.com/submariner-io/submariner/pkg/version/version.version=$(VERSION)"
- Remove main.VERSION
- Replace main.VERSION usage with version.version